field_points/Kenya/README.md

```diff
@@ -14,28 +14,57 @@
 <a title="TUBS, CC BY-SA 3.0 &lt;https://creativecommons.org/licenses/by-sa/3.0&gt;, via Wikimedia Commons" href="https://commons.wikimedia.org/wiki/File:Kenya,_administrative_divisions_-_de_-_colored.svg"><img width="300" alt="Kenya, administrative divisions - de - colored" src="https://upload.wikimedia.org/wikipedia/commons/thumb/5/54/Kenya%2C_administrative_divisions_-_de_-_colored.svg/512px-Kenya%2C_administrative_divisions_-_de_-_colored.svg.png"></a>
 
 ### Geography
+**Location** <sup>[1]</sup>
+- Eastern Africa
+- Bordering the Indian Ocean
+- Between Somalia & Tanzania
+
 **Area** <sup>[1]</sup>
 - Total: 580,367 sq km
 - Land: 569,140 sq km
 - Water: 11,227 sq km
 
-**Landscape**
-
+**Landscape** <sup>[3]</sup>
+1. Lake Victoria Basin: part of a plateau rising eastward from the lakeshore to the Rift highlands
+2. Great Rift Valley & Associated Highlands: splits the highland region into two sections: Mau Escarpment to the west & Aberdare Range to the east, valley itself is 30 to 80 miles (50 to 130 km) wide, important in historic & economic development, region was focus of European settlement
+3. Eastern Plateau Forelands: just east of Rift highlands, constitute a vast plateau of ancient rocks gently sloping to the coastal plain, scattered hills & striking elevated formations
+4. North & Northeast Semi-arid/Arid Regions: vast region extending from Ugandan border through Lake Rudolf to the plateau area between the Ethiopian and Kenyan highlands, movement of people & livestock is limited by availability of water
+5. Coastal Region: runs for about 250 miles (400 km) along the Indian Ocean, mangrove swamps, low-lying coastal areas
 
-**Terrestrial Ecoregions** 
-<!-- 1. Guinean forest-savanna mosaic
+**Terrestrial Ecoregions**
+1. Guinean forest-savanna mosaic
 2. Sahelian Acacia savanna
-3. West Sudanian savanna, and
-4. Guinean mangroves -->
+3. West Sudanian savanna
+4. Guinean mangroves
+
+**Land Use** <sup>[1]</sup>
```

Comment:
This is similar information to FAOStat below. 
I recommend creating a new header: "Land Use". 
and adding this and FAOStat land cover area and/or proportion information.
This information can go under subheadings: "FAO Stat" and the source for the below info.
Let me know if that makes sense!

Reply:
Alright, I edited the land use section - let me know if its right, or if I should change something/add more info for each of the ecoregion

Reply:
is FAO stat just a general term? I got the land use stats from this site https://www.cia.gov/the-world-factbook/countries/kenya/#geography

Reply:
The source for the information you added is the CIA Factbook, so that should be its heading.
The FAOStat data is at the bottom of the readme, you can just move it up and put it under the heading FAOStat.
This way we have the land area proportions from different sources beside each other